src/analysis/specific_limits.lean

```diff
@@ -110,17 +113,40 @@ lemma tendsto_pow_at_top_at_top_of_gt_1 {r : ℝ} (h : 1 < r) :
   ⟨n, λ m hnm, le_of_lt $
     lt_of_lt_of_le hn (pow_le_pow (le_of_lt h) hnm)⟩
 
+lemma tendsto_inverse_at_top_nhds_within_0 :
+  tendsto (λr:ℝ, r⁻¹) at_top (nhds_within 0 (set.Ioi 0)) :=
+begin
+  rw tendsto_at_top',
+  assume s hs,
+  rw mem_nhds_within_Ioi_iff_exists_Ioc_subset at hs,
+  rcases hs with ⟨C, C0, hC⟩,
+  refine ⟨C⁻¹, λ x hx, hC _⟩,
+  have : 0 < x, from lt_of_lt_of_le (inv_pos C0) hx,
+  exact ⟨inv_pos this, (inv_le C0 this).1 hx⟩
+end
+
+lemma tendsto_inverse_nhds_within_0_at_top :
+  tendsto (λ r:ℝ, r⁻¹) (nhds_within 0 (set.Ioi 0)) at_top :=
```

Comment:
Isn't this already contained in `tendsto_inv_zero_at_top` in the same file? Also, maybe you could use the more general typeclasses there for some of your statements.

Reply:
Thanks for catching this! Fixed.